static/js/src/entry/business/BusinessForm.vue

```diff
@@ -0,0 +1,393 @@
+        <div class="col">
+          <pay-fees
+            base-classes="form__fees"
+            amount-store-module="businessForm"
+            pay-fees-value-store-module="businessForm"
+            installment-period-store-module="businessForm"
+          />
+        </div>
+      </div>
+
+      <div class="grid_row">
+        <div class="col">
+          <native-pay
+            :form-is-valid="nativeIsValid"
+            :supported="nativeIsSupported"
+            base-classes="form__native"
+            amount-store-module="businessForm"
+            @setValue="setValue"
+            @onSubmit="onSubmit"
+          />
+        </div>
+      </div>
+
+      <div
+        v-if="nativeIsSupported && showManualPay"
+        class="grid_separator--l"
+        aria-hidden="true"
+      />
+
+      <div :aria-live="nativeIsSupported ? 'polite' : false">
+        <div v-if="showManualPay">
+          <div class="grid_row">
+            <div class="col">
+              <manual-pay
+                :show-error="showManualErrors"
+                :validation="validation.card"
+                base-classes="form__manual"
+                @setValidationValue="setValidationValue"
+              />
+            </div>
+          </div>
+
+          <div class="grid_row">
+            <div class="col">
+              <manual-submit
+                :form-is-valid="manualIsValid"
+                :is-fetching-token="isFetchingToken"
+                base-classes="form__submit button button--yellow button--l"
+                value="Donate"
+                @onSubmit="onSubmit"
+                @setValue="setValue"
+                @setValidationValue="setValidationValue"
+              />
+            </div>
+          </div>
+        </div>
+        <p class="subtext">The Texas Tribune is a 501(c)(3), and your organization's gift is tax deductible</p>
+      </div>
+
+      <div
+        v-if="showErrorClue"
+        class="grid_separator--l"
+        aria-hidden="true"
+      />
+
+      <div
+        v-if="showErrorClue"
+        class="grid_row"
+        aria-hidden="true"
+      >
+        <div class="col">
+          <p class="form__error form__error--normal form__error--centered">Please correct errors above</p>
+        </div>
+      </div>
+
+      <local-hidden
+        :value="stripeToken"
+        name="stripeToken"
+      />
+      <hidden
+        name="campaign_id"
+        store-module="businessForm"
+      />
+      <hidden
+        name="referral_id"
+        store-module="businessForm"
+      />
+      <hidden
+        name="installments"
+        store-module="businessForm"
+      />
+      <hidden
+        name="description"
+        store-module="businessForm"
+      />
+      <hidden
+        name="pay_fees_value"
+        store-module="businessForm"
+      />
+      <hidden
+        name="openended_status"
+        store-module="businessForm"
+      />
+      <hidden
+        name="customerId"
```

Comment:
Should be able to remove this.

Reply:
Okay,. Makes sense, not being used. Will clean it out.